src/add-to-app/android/project-setup.md

```diff
@@ -172,7 +200,8 @@ The source code subproject mechanism is a convenient
 one-click build process, but requires the Flutter SDK.
 This is the mechanism used by the Android Studio IDE plugin.
 
-#### Option A - Depend on the Android Archive (AAR)
+### Option A - Depend on the Android Archive (AAR)
```

Comment:
Shouldn't "Option A and Option B" use another set of 2 tabs? It seems odd to diverge from tabs after using them above.

Also, I don't see the text here, but there are a couple of Notes in the staged site that talk about .android and .androidPackage files. I assume these notes only apply to Mac. What about Windows?

Reply:
1. Changed to tabs.
2. The `.android` directory is a local version of the full Android OS for the emulator. It exists in Windows, too.

Reply:
Interesting. Dot files are generally a Unix-only convention.